opengrok-tools/src/main/python/opengrok_tools/all/utils/command.py

```diff
@@ -169,6 +174,9 @@ def run(self):
 
                     self.out.append(line)
 
+                    if self.doprint:
+                        print(line.strip())
```

Comment:
I'm little bit worried that while the intention is clear (to remove the trailing `\n`) this will destroy formatted messages with `tab` or `spaces` at the beginning of the line. I can imagine for example exception stacktraces which are indented in the log.

Could you please check it?

Reply:
of course I wanted to use rstrip()